101-automation-with-modules-and-runbook-execution/azuredeploy.json

```diff
@@ -0,0 +1,224 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "UserEmail": {
+      "type": "string",
+      "metadata": {
+        "description": "azuread email"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "allowedValues": [
+        "japaneast",
+        "eastus2",
+        "westeurope",
+        "southeastasia",
+        "southcentralus",
+        "koreacentral",
+        "westus2",
+        "brazilsouth",
+        "uksouth",
+        "westcentralus",
+        "northeurope",
+        "canadacentral",
+        "australiasoutheast",
+        "centralindia"
+      ],
+      "metadata": {
+        "description": "location allowed for automation account."
+      }
+    },
+    "Password": {
+      "type": "securestring",
+      "metadata": {
+        "description": "aduser password"
+      }
+    },
+	 "WebAppName": {
+      "type": "string",
+      "metadata": {
+        "description": "This is the name of the web app that may be provided dynamically after creating."
```

Comment:
is this web app being created or does it need to exist prior to deployment?

Reply:
It is existing webapp for which user want to update the app setting